client/me/purchases/manage-purchase/test/payment-info-block.js

```diff
@@ -309,4 +309,13 @@ describe( 'PaymentInfoBlock', () => {
 			} );
 		} );
 	} );
+	it( 'shows warning if auto-renew is enabled without a payment method', () => {
+		const purchase = {
+			isAutoRenewEnabled: 'true',
```

Comment:
Should this not be `true`?
```suggestion
			isAutoRenewEnabled: true,
```